src/Components/MediaPresenter.ts

```diff
@@ -195,3 +195,5 @@ export class MediaPresenter {
     );
   }
 }
+
+const mediaPresenter = new MediaPresenter();
```

Comment:
This isn't exported or used in this file. You could either remove this line or export it and import the instance in `App.tsx`